### ISSUE TITLE
epgen integration

### ISSUE THREAD

**Issue by chicoxyzzy** [littledan/javascript-standard-library-incubator#1]
I created https://github.com/chicoxyzzy/epgen (**E**CMAScript **P**roposal **Gen**erator) recently. It's not documented well yet, but it can make things a bit easier. Does it make sense to add stdlib template to epgen?

**Comment by littledan** [comment]
Yeah, that could be great. Maybe a somewhat edited version could replace https://github.com/chicoxyzzy/epgen/blob/master/templates/base/README.md ?
